lib/plausible_web/controllers/stats_controller.ex

```diff
@@ -105,30 +161,28 @@ defmodule PlausibleWeb.StatsController do
     |> send_resp(200, zip_content)
   end
 
+  @doc """
+    See: https://plausible.io/docs/shared-links
+
+    Authorizes and renders a shared link:
+    1. Shared link with no password protection: needs to just make sure the shared link entry is still
+    in our database. This check makes sure shared link access can be revoked by the site admins. If the
+    shared link exists, render it directly.
+
+    2. Shared link with password protection: Same checks as without the password, but an extra step is taken to
+    protect the page with a password. When the user passes the password challenge, a cookie is set with Plausible.Auth.Token.sign_shared_link().
```

Comment:
This makes ex_doc link the function.

```suggestion
    protect the page with a password. When the user passes the password challenge, a cookie is set with `Plausible.Auth.Token.sign_shared_link/0`.
```